approval_frame/forms.py

```diff
@@ -17,3 +18,22 @@ class RegistrationFormCustom(RegistrationFormUniqueEmail):
         error_messages={'invalid': _("This value may contain only letters, "
             + "numbers and ./+/-/_ characters.")}
     )
+
+class PasswordResetFormCustom(PasswordResetForm):
```

Comment:
I didn't know that about Github! Looks like they're not following good security measures.
And you've already added the feature that allows them to login using their email ID :smile: 
So we don't have to worry about them forgetting their username.